src/app/tabs/newAdmin/textrecords/components/TextRecordInputComponent.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { multilanguage } from 'redux-multilanguage';
+import {
+  Row, Col, Button,
+} from 'react-bootstrap';
+
+const TextRecordInputComponent = ({ strings, handleClick, disabled }) => {
+  const [state, setState] = useState({
+    key: '',
+    value: '',
+    encodings: {
+      text: true,
+    },
+  });
+  return (
+    <Row className="textRecordInput">
+      <Col md="12">
+        <label htmlFor="key">
+          Key
+          <input type="text" list="key" onChange={e => setState({ ...state, key: e.target.value })} />
+          <datalist
+            id="key"
+            onChange={e => setState({ ...state, key: e.target.value })}
+          >
+            <option value="email">email</option>
+            <option value="url">url</option>
+            <option value="avatar">avatar</option>
+            <option value="description">description</option>
+            <option value="notice">notice</option>
+            <option value="keywords">keywords</option>
+            <option value="com.discord">com.discord</option>
+            <option value="com.github">com.github</option>
+            <option value="com.reddit">com.reddit</option>
+            <option value="com.twitter">com.twitter</option>
+            <option value="org.telegram">org.telegram</option>
```

Comment:
You could dynamically create these based on the list that will be single sourced from the opperations.js file.